res/controllers/midi-components-0.0.js

```diff
@@ -765,12 +763,19 @@
 
     JogWheelBasic.prototype = new Component({
         vinylMode: true,
+        get hasDeckNumber() {
+            return Number.isInteger(this.deck) && this.deck > 0;
+        },
```

Comment:
we haven't used js getters until now. have you made sure this works? I'm not confident the Object merging logic handles getters and setters correctly.

Reply:
You're right. I had to use `Object.defineProperties` instead.